src/components/utils/version.ts

```diff
@@ -0,0 +1,53 @@
+// copied from Azure Data Explorer plugin
```

Comment:
it doesn't look like these are exported except for use by the divider component. can this file also be moved to a draft branch and then linked to the github issue for enabling the divider component?

Reply:
Good point, removed!